client/build.gradle

```diff
@@ -8,11 +8,7 @@ plugins {
     id 'org.hidetake.swagger.generator'
 }
 
-dependencyManagement {
-    imports {
-        mavenBom(SpringBootPlugin.BOM_COORDINATES)
-    }
-}
+repositories { mavenCentral() }
```

Comment:
not related?

Reply:
trying to make the pom generation happy :(